src/frontend/data/sideEffects/upload/index.ts

```diff
@@ -66,10 +67,25 @@ export const upload = (
       notifyObjectUploadProgress,
     );
 
-    await addResource(objectType, {
-      ...object,
-      upload_state: uploadState.PROCESSING,
-    });
+    if (object.hasOwnProperty('title')) {
+      await addResource(objectType, {
```

Comment:
I think we need a comment to explain why we `addResource` then `updateResource` because from the outside it looks redundant